modules/init.gradle

```diff
@@ -6,6 +6,14 @@ import org.gradle.internal.hash.HashValue
 if (System.properties["http.proxyHost"] == "squid.lax.liferay.com") {
 	logger.lifecycle "Using proxy {}:{}.", System.properties["http.proxyHost"], System.properties["http.proxyPort"]
 
+	File repositoryLocalDir = file("../.m2-tmp")
+
+	URI uri = repositoryLocalDir.toURI()
+
+	String repositoryLocalUrl = uri.toASCIIString()
+
+	System.setProperty "repository.local.url", repositoryLocalUrl
```

Comment:
Is it correct to set this system property only on CI? After all, only CI uses our Squid proxy.